products/postgresql.md

```diff
@@ -27,32 +27,32 @@ identifiers:
 releases:
 -   releaseCycle: "15"
     eol: 2027-11-11
-    latest: "15.2"
-    latestReleaseDate: 2023-02-06
+    latest: "15.3"
```

Comment:
As always, git tag has been pushed but those versions are not released officially. I can't see a convenient way to fix this though.